login-workflow/src/translations/chinese.ts

```diff
@@ -27,6 +27,7 @@ export const pxbAuthChineseTranslations: PxbAuthTranslations = {
         DEBUG_MODE: '调试模式',
         FORGOT_PASSWORD_LINK: '[测试忘记密码的电子邮件]',
         TEST_INVITE_LINK: '[测试邀请注册]',
+        INCORRECT_CREDENTIALS: '邮箱地址或密码错误',
```

Comment:
We call this `INVALID_CREDENTIAL` on the react side if you wan't some parity there. Same for the other 3 translations... not going to write this comment 4 times though.

Reply:
Ahh, you'll have to update the type definition too if you change this.

Reply:
Good catch, this is updated now.